pkg/operator/encryption/controllers.go

```diff
@@ -101,10 +103,10 @@ type Controllers struct {
 	controllers []runner
 }
 
-func (c *Controllers) Run(stopCh <-chan struct{}) {
+func (c *Controllers) Run(ctx context.Context, workers int) {
```

Comment:
just wondering if we want to expose `workers` IIRC all `encryption` controllers are single-threaded.